feature-flags/index.js

```diff
@@ -0,0 +1,118 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const FLAGS_PREFIX = 'featureFlags:';
+const FLAGS_KEY = 'current';
+
+module.exports = initialise;
+
+/**
+ * Initialise the feature-flag module.
+ *
+ * @param {Object} config - Configuration object.
+ * @param {Object} config.interval - Refresh interval.
+ * @param {Object} [config.redis] - Redis config.
+ * @param {String} [config.redis.host] - Redis host name or IP address.
+ * @param {Number} [config.redis.port] - Redis port.
+ * @param {Number} [config.redis.maxConnections] - Maximum number of connections in Redis pool.
+ * @param {Number} [config.redis.maxPending] - Maximum number of clients waiting for a connection.
+ * @param {Number} [config.redis.minConnections] - Minimum number of clients in Redis pool.
+ *
+ * @param {Object} log - Log object.
+ *
+ * @param {Object} [defaults] - Default experiment state to return in the event of error.
+ *                              If not set, calls to FeatureFlags.get may fail.
+ *
+ * @returns {FeatureFlags}
+ */
+function initialise (config, log, defaults) {
+  const { interval, redis: redisConfig } = config;
+
+  if (! (interval > 0 && interval !== Infinity)) {
```

Comment:
For some reason `<= Infinity` read a little cleaner to me? Also, does it makes sense here to have a default refresh interval and make it optional.

Reply:
I'll update the `Infinity` comparison, thanks. Not feeling persuaded by the default refresh interval though, because we already have defaults in config so I don't see it ever being used.